oak_channel/src/lib.rs

```diff
@@ -37,6 +37,9 @@ use anyhow::Context;
 use bytes::BytesMut;
 use oak_core::timer::Timer;
 
+/// Simple no_std compatible equivalent of [`std::io::Read`].
+///
+/// [`std::io::Read`]: <https://doc.rust-lang.org/std/io/trait.Read.html>
 pub trait Read {
```

Comment:
interestingly rust doc does not get these links by default. Perhaps related to the building docs from a crate that declares #![no_std]